canaries/pcengines/canary-001-2019.txt.sig.piotr-krol

```diff
@@ -0,0 +1,69 @@
+-----BEGIN PGP MESSAGE-----
```

Comment:
@pietrushnic why is Your signature a PGP message?

Reply:
@miczyg1 hmm, good question, I have not idea

Reply:
@pietrushnic maybe if You repeat the signing procedure with fixed command, it will change